modules/TestUtils.js

```diff
@@ -88,6 +94,14 @@ export const functionThrows = (fn, context, args, value) => {
 export const arrayContains = (array, value, compareValues) =>
   array.some(item => compareValues(item, value) !== false)
 
+/**
+ * Returns true if the given array contains the value, false
+ * otherwise. The compareValues function must return false to
+ * indicate a non-match.
+ */
+export const setContains = (set, value, compareValues) =>
+  arrayContains([ ...set ], value, compareValues)
```

Comment:
`Set#has` is using identity equality (`===`) so `new Set([ {} ]).has({})` is false.